### DISCUSSION TITLE
VSCode Intellisense is not suggesting import some methods in `@jest/globals`

### DISCUSSION THREAD

**Discussion by viceice** [orta/vscode-jest#857] [Q&A]
I currently struggle, why vscode isn't suggesting to  import some methods  from  `@jest/globals`.

It works for `beforeEach` but not for `test` or `it`

![image](https://user-images.githubusercontent.com/1798109/165900770-5e6b5ed9-88b5-4d80-a333-cc18f317a564.png)
![image](https://user-images.githubusercontent.com/1798109/165900831-9ff2665c-8a5f-441e-a379-8943b30f5185.png)

It also works when simply typing `test`
![image](https://user-images.githubusercontent.com/1798109/165901081-38c8d85a-2297-4317-8fa8-823e074f1168.png)

**Scenario**
We have a [`nx` ](https://github.com/nrwl/nx) angular workspace with jest and need to run `jest` in `ESM` mode.
In that mode not all `jest` variables are globally available and needs to be imported via `@jest/globals`. So we explicit removed `@types/jest` from `tsconfig.json`, so we don't suggest to use the globals.

You can see in screenshots above, that it works fine for `beforeEach` but not for `it` or `test` functions.

**Comment by connectdotz** [comment]
did you get a chance to check out #686?

**Reply by viceice** [reply]
That's a different use case. I'll update my description.

**Reply by viceice** [reply]
I can also create a minimal reproduction repo to show the issue
